lib/widgets/body/how_to_play/how_to_play_part_two.dart

```diff
@@ -0,0 +1,154 @@
+  late final Animation<Offset> _animation;
+
+  @override
+  void initState() {
+    super.initState();
+    _animationController =
+        AnimationController(vsync: this, duration: const Duration(seconds: 2));
+    _animation = Tween<Offset>(
+      begin: const Offset(300, 0.0),
+      end: const Offset(120, 0.0),
+    )
+        .chain(CurveTween(curve: Curves.easeOutQuart))
+        .animate(_animationController);
+    widget.scrollNotifier.addListener(this);
+  }
+
+  @override
+  void dispose() {
+    _animationController.dispose();
+    widget.scrollNotifier.deleteListener(this);
+    super.dispose();
+  }
+
+  @override
+  void update(double subject) {
+    if (!_animationController.isAnimating && subject > widget.startingPoint) {
+      _animationController.forward();
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      alignment: Alignment.topCenter,
+      children: [
+        Positioned(
+          top: 822.0,
+          child: Padding(
+            padding: const EdgeInsets.only(right: 4.0),
+            child: Image.asset(
+              'images/line_2.png',
+              color: NyxsColors.mint,
+              height: 484.1,
+              fit: BoxFit.fitHeight,
+              filterQuality: FilterQuality.high,
+            ),
+          ),
+        ),
+        Positioned(
+          top: widget.imageTop,
+          child: AnimatedBuilder(
+            animation: _animation,
+            child: Padding(
+              padding: const EdgeInsets.only(left: 33.0),
+              child: Image.asset(
+                'images/player_cards.png',
+                height: 137.0,
+                fit: BoxFit.fitHeight,
+                filterQuality: FilterQuality.high,
+              ),
+            ),
+            builder: (context, child) {
+              return Transform.translate(
+                offset: _animation.value,
+                child: Opacity(
+                  opacity: _animationController.value,
+                  child: child,
+                ),
+              );
+            },
+          ),
+        ),
+        my.AnimatedWidget(
+          scrollNotifier: widget.scrollNotifier,
+          position: widget.decorationTop,
+          startingPoint: 1763.0 + 80,
+          reverse: true,
+          child: Padding(
+            padding: const EdgeInsets.only(left: 138.0),
+            child: Image.asset(
+              'images/decoration_2.png',
+              width: 152.0,
+              height: 246.0,
+              fit: BoxFit.fill,
+              filterQuality: FilterQuality.high,
+            ),
+          ),
+        ),
+        my.AnimatedWidget(
+          scrollNotifier: widget.scrollNotifier,
+          position: widget.descriptionTop,
+          startingPoint: 1550.0,
+          child: Stack(
+            children: [
+              const Positioned(
+                child: Padding(
+                  padding: EdgeInsets.only(
+                    left: 185.0,
```

Comment:
Can you modify the left padding? 
I changed the font before, maybe that's why there's a difference.

expect: 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/60090790/196121976-ff180a03-a208-427b-a3bf-f15021f66f73.png">

now: 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/60090790/196122692-99e463c9-c2fc-474a-9bb5-3051ce4b6286.png">

when value is 187:
<img width="300" alt="Screen Shot 2022-10-17 at 3 03 57 PM" src="https://user-images.githubusercontent.com/60090790/196122596-4c07a075-9936-4a88-9f64-d6cafe9a79cf.png">

Reply:
I changed it!